src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -248,27 +328,111 @@ abs_request_creator::make_list_blobs_request(
     return header;
 }
 
+result<http::client::request_header>
+abs_request_creator::make_get_account_info_request() {
+    const boost::beast::string_view target
+      = "/?restype=account&comp=properties";
+    const boost::beast::string_view host{_ap().data(), _ap().length()};
+
+    http::client::request_header header{};
+    header.method(boost::beast::http::verb::head);
+    header.target(target);
+    header.insert(boost::beast::http::field::host, host);
+
+    auto error_code = _apply_credentials->add_auth(header);
+    if (error_code) {
+        return error_code;
+    }
+
+    return header;
+}
+
+result<http::client::request_header>
+abs_request_creator::make_delete_file_request(
+  const access_point_uri& adls_ap,
+  bucket_name const& name,
+  object_key const& path) {
+    // DELETE /{container-id}/{path} HTTP/1.1
+    // Host: {storage-account-id}.dfs.core.windows.net
+    // x-ms-date:{req-datetime in RFC9110} # added by 'add_auth'
+    // x-ms-version:"2021-08-06"           # added by 'add_auth'
+    // Authorization:{signature}           # added by 'add_auth'
+    const auto target = fmt::format("/{}/{}", name(), path().string());
+
+    const boost::beast::string_view host{adls_ap().data(), adls_ap().length()};
+
+    http::client::request_header header{};
+    header.method(boost::beast::http::verb::delete_);
+    header.target(target);
+    header.insert(boost::beast::http::field::host, host);
+
+    header.set("x-ms-version", "2023-01-03");
```

Comment:
I'm curious how this was picked, vs going with the latest version. Though I imagine it only matters if we want to use new features, given their supported versions go way back.

Reply:
`2023-01-03` is the latest version. The version is is set when we call `add_auth`. Having this here is a leftover, from some other stuff I tried. Will remove.

Reply:
Fixed.

Reply:
That's the right page. They updated it since I last looked at it. I don't think there's much point playing catch-up
with their api versions unless we want a specific feature introduced by a newer version.